README.md

```diff
@@ -294,6 +294,14 @@ Field overrides allow you to define custom GraphQL types or resolvers on top of
 
 for more information in Overrides, they adhere to the GraphQLFieldConfig outlined here https://graphql-compose.github.io/docs/basics/what-is-resolver.html
 
+## Advanced Config
+
+### `fieldNameTransform`
+
+Accepts a function which takes in field names and transforms them for the GraphQL schema generation -- this is used internally to remove spaces but can be used for other global transforms as well
+
+// some example here
```

Comment:
We should replace this with an example of end-user config usage

Reply:
lol didn't realize I'd forgotten to come back to that